e2etest/rewarding_test.go

```diff
@@ -572,48 +589,44 @@ func updateExpectationWithPendingClaimList(
 }
 
 func newConfig(
-	chainDBPath,
-	trieDBPath,
-	indexDBPath string,
 	producerPriKey crypto.PrivateKey,
 	networkPort,
 	apiPort int,
-	numNodes uint64,
+	HTTPAdminPort int,
 ) config.Config {
 	cfg := config.Default
 
+	cfg.Plugins[config.GatewayPlugin] = true
+	cfg.Chain.EnableAsyncIndexWrite = false
+
+	cfg.System.HTTPAdminPort = HTTPAdminPort
 	cfg.Network.Port = networkPort
 	cfg.Network.BootstrapNodes = []string{"/ip4/127.0.0.1/tcp/4689/ipfs/12D3KooWJwW6pUpTkxPTMv84RPLPMQVEAjZ6fvJuX4oZrvW5DAGQ"}
 
 	cfg.Chain.ID = 1
-	cfg.Chain.ChainDBPath = chainDBPath
-	cfg.Chain.TrieDBPath = trieDBPath
-	cfg.Chain.IndexDBPath = indexDBPath
 	cfg.Chain.CompressBlock = true
 	cfg.Chain.ProducerPrivKey = producerPriKey.HexString()
-	cfg.Chain.EnableAsyncIndexWrite = false
 
 	cfg.ActPool.MinGasPriceStr = big.NewInt(0).String()
 
 	cfg.Consensus.Scheme = config.RollDPoSScheme
-	cfg.Consensus.RollDPoS.FSM.UnmatchedEventInterval = 120 * time.Millisecond
-	cfg.Consensus.RollDPoS.FSM.AcceptBlockTTL = 200 * time.Millisecond
-	cfg.Consensus.RollDPoS.FSM.AcceptProposalEndorsementTTL = 100 * time.Millisecond
-	cfg.Consensus.RollDPoS.FSM.AcceptLockEndorsementTTL = 100 * time.Millisecond
-	cfg.Consensus.RollDPoS.FSM.CommitTTL = 100 * time.Millisecond
+	cfg.Consensus.RollDPoS.FSM.UnmatchedEventInterval = 2400 * time.Millisecond
+	cfg.Consensus.RollDPoS.FSM.AcceptBlockTTL = 1800 * time.Millisecond
+	cfg.Consensus.RollDPoS.FSM.AcceptProposalEndorsementTTL = 1800 * time.Millisecond
+	cfg.Consensus.RollDPoS.FSM.AcceptLockEndorsementTTL = 1800 * time.Millisecond
+	cfg.Consensus.RollDPoS.FSM.CommitTTL = 600 * time.Millisecond
 	cfg.Consensus.RollDPoS.FSM.EventChanSize = 100000
 	cfg.Consensus.RollDPoS.ToleratedOvertime = 1200 * time.Millisecond
 	cfg.Consensus.RollDPoS.Delay = 6 * time.Second
 
 	cfg.API.Port = apiPort
 
-	cfg.Genesis.Blockchain.NumSubEpochs = 4
+	cfg.Genesis.BlockInterval = 6 * time.Second
+	cfg.Genesis.Blockchain.NumSubEpochs = 2
 	cfg.Genesis.Blockchain.NumDelegates = numNodes
 	cfg.Genesis.Blockchain.TimeBasedRotation = true
-	cfg.Genesis.Delegates = cfg.Genesis.Delegates[0:numNodes]
-
-	cfg.Genesis.BlockInterval = 500 * time.Millisecond
-	cfg.Genesis.EnableGravityChainVoting = true
-	cfg.Genesis.Rewarding.FoundationBonusLastEpoch = 2
+	cfg.Genesis.Delegates = cfg.Genesis.Delegates[3 : numNodes+3]
+	cfg.Genesis.EnableGravityChainVoting = false
+	cfg.Genesis.PollMode = "lifeLong"
```

Comment:
can you explain what is the root-cause and fix?

saw a lot of params changed like 
1. adding "lifelong"
2. EnableGravityChainVoting changed to false
3. removing FoundationBonusLastEpoch = 2
4. block interval and TTL value change to seconds, but these are local tests, the original ~200ms value should allow the test to pass

Reply:
the config copy form other place.
if we don't "lifeLong"，got error "fail to create chain service: failed to generate poll protocol: unsupported poll mode"